### PR TITLE
Add Jfrog Artifactory support

### PR DESCRIPTION
**Pull request by steinbrueckri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,72 @@
...
 ```
 
+Create ```settings.xml``` with [Jfrog Artifactory](https://www.jfrog.com/confluence/display/JFROG/Maven+Repository) in repository list
+
+```yml
+steps:
+- uses: s4u/maven-settings-action@v2.0
+  with:
+    servers: '[{"id": "central", "username": "username", "password": "password"}]'
+    servers: '[{"id": "snapshots", "username": "username", "password": "password"}]'
```
Comment (slawekjaranowski) [inline README.md:55]: I guess credentials for both artifactory repository id `central` and `snapshot` will be the same.
So maybe better add params: `artifactory_username`, `artifactory_password` and in code create two servers item for theam.

File: action.yml [hunk 2]
```diff
@@ -16,6 +16,10 @@ inputs:
     description: 'add https://oss.sonatype.org/content/repositories/snapshots to repository list - true or false'
     default: "false"
     required: false
+  artifactory_org:
+    description: 'enable jfrog default profiles and set jfrog artifactory org name'
+    default: "myorg"
```
Comment (slawekjaranowski) [inline action.yml:21]: What happens when user not provide settings for this, I think this value will be used and artifactory feature will be configured in settings.xml
Better will be without default.

File: dist/settings.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+               <releases>
+                  <enabled>false</enabled>
+               </releases>
+               <snapshots>
+                  <enabled>true</enabled>
+               </snapshots>
+            </pluginRepository>
+         </pluginRepositories>
+      </profile>
+      <!-- mms ################################################################## -->
```
Comment (slawekjaranowski) [inline dist/settings.xml:44]: ?

File: action.yml [hunk 4]
```diff
@@ -29,5 +33,4 @@ inputs:
 
 runs:
   using: 'node12'
-  main: 'index.js'
-  post: 'cleanup.js'
```
Comment (slawekjaranowski) [inline action.yml:33]: why you remove `post` action?

**Comment by steinbrueckri** [comment]
@slawekjaranowski I will close this PR and come back later when I fix this ...

**Comment by slawekjaranowski** [comment]
ok, I will be waiting.

**Comment by slawekjaranowski** [comment]
Please create issue for your idea to inform other what going.

**Comment by slawekjaranowski** [comment]
And one another case: please consider to use mirror instead of override repository of id central

https://issues.apache.org/jira/browse/MNG-6879

**Comment by steinbrueckri** [comment]
@slawekjaranowski mirrors are also a good idea but I need repositories because I want to deploy a jar to a jfrog.

**Comment by slawekjaranowski** [comment]
IHMO for deployment we use `distributionManagement` in pom file not `repository`

http://maven.apache.org/plugins/maven-deploy-plugin/usage.html

**Comment by steinbrueckri** [comment]
new PR #12 :)

**Pull request closed without merging**